internal/controllers/machineset/machineset_controller.go

```diff
@@ -352,10 +350,10 @@ func (r *Reconciler) syncReplicas(ctx context.Context, ms *clusterv1.MachineSet,
 	switch {
 	case diff < 0:
 		diff *= -1
-		log.Info("Too few replicas", "need", *(ms.Spec.Replicas), "creating", diff)
+		log.Info(fmt.Sprintf("MachineSet is scaling up to %d replicas by creating %d machines", *(ms.Spec.Replicas), diff), "replicas", *(ms.Spec.Replicas), "machineCount", len(machines))
```

Comment:
I'm not sure if the k/v pairs are helpful for the reader (i.e. not sure if it's clear that replicas is replicas from spec and machineCount is the number of currently existing machines (+ some filtering applied (?)).

Maybe the string is already enough. Given it contains almost the same data and we won't really use replicas and machineCount to filter the logs later

But feel free to just resolve if you want to keep it as is

Reply:
Let's keep it for now